docs/_docs/sql-reference/ddl.adoc

```diff
@@ -245,6 +245,60 @@ Drop Person table if the one exists:
 DROP TABLE IF EXISTS "Person";
 ----
 
+== CREATE INDEX
+
+Creates a new index.
+
+[source,sql]
+----
+CREATE INDEX [IF NOT EXISTS] name ON tableName
+[USING { HASH | TREE }]
+(column_name [ASC | DESC]
+[NULLS {FIRST | LAST}]);
+----
+
+Parameters:
+
+* `name` - name of the index.
+* `tableName` - name of the table to create the index on.
+* `USING` - specifies whether the command creates a B+ Tree index or a hash index. Possible values: `HASH`, `TREE`.
+* `column_name` - specifies whether column names will be sorted in ascending or descending order. Possible values: `ASC`, `DESC`.
```

Comment:
this line doesn't make any sense. `column_name` specifies which column of table of origin should be indexed, whereas `ASC` and `DESC` specifies in which order this column should be sorted